crates/matrix-sdk/examples/autojoin.rs

```diff
@@ -49,13 +49,13 @@ async fn login_and_sync(
         // The location to save files to
         let mut home = dirs::home_dir().expect("no home directory found");
         home.push("autojoin_bot");
-        let state_store = matrix_sdk_sled::StateStore::open_with_path(home)?;
+        let state_store = matrix_sdk_sled::StateStoreBuilder::default().path(home).build()?;
```

Comment:
```suggestion
        let state_store = matrix_sdk_sled::StateStore::builder().path(home).build()?;
```

same below and in the other example(s).